Examples/MAX32650/SPI/main.c

```diff
@@ -221,5 +221,5 @@ int main(void)
         LED_On(1);
     }
 
-    return E_NO_ERROR;
+    return 0;
```

Comment:
We should keep `return E_NO_ERROR` instead of using `return 0` for all main functions